spec/index.html

```diff
@@ -1681,7 +1691,7 @@ <h2>Internet Media Type, File Extension and Macintosh File Type</h2>
 
     <dt>Encoding considerations:</dt>
     <dd>The syntax of Turtle is expressed over code points in Unicode [[!UNICODE]]. The encoding is always UTF-8 [[!UTF-8]].</dd>
-    <dd>Unicode code points may also be expressed using an \uXXXX (U+0000 to U+FFFF) or \UXXXXXXXX syntax (for U+10000 onwards) where X is a hexadecimal digit [0-9A-Fa-f]</dd>
+    <dd><a data-cite="i18n-glossary#dfn-code-point">Unicode code points</a> may also be expressed using <code>\uXXXX</code> (<code class="codepoint">U+0000</code> to <code class="codepoint">U+FFFF</code>) or <code>\UXXXXXXXX</code> syntax (for <code class="codepoint">U+10000</code> onwards) where X is a hexadecimal digit `[0-9A-Fa-f]`</dd>
```

Comment:
Is there anything that prevents to write for example `\U00000000` (i.e. encode codpoints lower than U+10000 using 8 hex chars?

Reply:
```suggestion
    <dd><a data-cite="i18n-glossary#dfn-code-point">Unicode code points</a> may also be expressed
      using <code>\uXXXX</code> (<code class="codepoint">U+0000</code> to <code class="codepoint">U+FFFF</code>)
      or <code>\UXXXXXXXX</code> syntax (for code points up to <code class="codepoint">U+10FFFF</code>)
      where X is a hexadecimal digit `[0-9A-Fa-f]`</dd>
```